__tests__/app.test.js

```diff
@@ -36,16 +54,79 @@ describe("GET /api/topics", () => {
     });
 });
 
-describe("GET /api", () => {
-    it("200: responds with an object describing all the available endpoints on API", () => {
+//Articles
+
+describe("GET /api/articles", () => {
+    it("200: responds with all articles sorted by date in descending order by default", () => {
         return request(app)
-            .get("/api")
+            .get("/api/articles")
             .expect(200)
             .then((res) => {
-                expect(res.body).toEqual(endpoints)
+                expect(Array.isArray(res.body.articles)).toBe(true);
+                expect(res.body.articles.length).toBe(13);
+                res.body.articles.forEach(article => {
+                    expect(article).toHaveProperty('author');
+                    expect(article).toHaveProperty('title');
+                    expect(article).toHaveProperty('article_id');
+                    expect(article).toHaveProperty('body');
+                    expect(article).toHaveProperty('topic');
+                    expect(article).toHaveProperty('created_at');
+                    expect(article).toHaveProperty('votes');
+                    expect(article).toHaveProperty('article_img_url');
+                })
+                expect(res.body.articles).toBeSorted({ key: 'created_at', descending: true });
             })
-    })
-})
+    });
+    it("200: responds with all articles sorted by date in descending order omitting the body property", () => {
+        return request(app)
+            .get("/api/articles?properties=author,title,article_id,topic,created_at,votes,article_img_url")
```

Comment:
We shouldn't have to put a query in here. By just going to `/api/articles`, the code should be written so that the response automatically comes back sorted by date in descending order, without the user having to pass anything else in,